.circleci/config.yml

```diff
@@ -164,14 +164,19 @@ jobs:
           name: Setup dependencies
           command: |
             apt-get update
-            apt-get install -qy ca-certificates clang
-      - attach_workspace:
-          at: /FISCO-BCOS/build/bin
+            apt-get install -y git clang make curl build-essential cmake libssl-dev zlib1g-dev
+      - checkout
+      - run:
+          name: Compile
+          command: |
+            mkdir -p build && cd build
+            CC=/usr/bin/clang CXX=/usr/bin/clang++ cmake -DSTATIC_BUILD=ON -DCMAKE_BUILD_TYPE=Release ..
+            make -j2
       - run:
           name: Strip
           command: |
             cd build/bin
-            strip fisco-bcos && rm fisco-bcos.tar.gz
+            strip fisco-bcos
```

Comment:
TODO: compile a real `Release` version for outside-users to in case of coredump caused by `assert`

Reply:
yes, this modify is try to complete what you want